### PR TITLE
video icon now white

### PR DESCRIPTION
**Pull request by blongden73**
Get some white on that Icon. 

@akash1810 @johnduffell

### CODE REVIEW

File: static/src/stylesheets/base/_inline-svgs.scss [hunk 1]
```diff
@@ -52,7 +52,7 @@
     }
 
     .tonal--tone-special-report & {
-        fill: $news-support-6;
+        fill: #ffffff;
```
Comment (akash1810) [inline static/src/stylesheets/base/_inline-svgs.scss]: could we use a variable? [`neutral-7`](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/palette/_src.scss#L15) perhaps? or a declare a new one?

**Comment by akash1810** [comment]
may be better to use a variable, other than that 👍

**Comment by akash1810** [comment]
👍

**Comment by johnduffell** [comment]
thanks ben!

**Pull request merged**